### PR TITLE
Unit tests

### PR DESCRIPTION
**Pull request by Jayakrishnan-MS**
Test coverage with above 80%

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -83,24 +97,21 @@
...
     "color": "4.2.3",
     "nullthrows": "1.1.1",
     "prosemirror-commands": "1.1.5",
+    "prosemirror-model": "~1.16.0",
     "prosemirror-dropcursor": "1.3.5",
     "prosemirror-gapcursor": "^1.2.2",
     "prosemirror-history": "1.1.3",
     "prosemirror-inputrules": "1.1.3",
     "prosemirror-utils": "1.0.0-0",
+    "@modusoperandi/licit-doc-attrs-step": "^0.1.6",
```
Comment (Tezra) [inline package.json:110]: needs to be ^0.1.5

File: src/HeadingCommand.test.ts [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  });
+  it('should enable the command when text style mark is enabled', () => {
+    const state = EditorState.create({ schema: schema1 });
+    const isEnabled = command.isActive(state);
+    expect(isEnabled).toBe(true);
+  });
+
+  it('execute without dispatch', () => {
+    const state = EditorState.create({ schema: schema1 });
+    command.execute(state);
```
Comment (Tezra) [inline src/HeadingCommand.test.ts:63]: missing expect

File: src/IndentCommand.test.ts [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+});
+
+
+
+
+
+
+
+
+
```
Comment (Tezra) [inline src/IndentCommand.test.ts:60]: way too much extra white space

File: src/TextAlignCommand.test.ts [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+
+  it('should enable the command when text align is enabled', () => {
+    const state = EditorState.create({ schema: schema1 });
+    const isEnabled = command.isActive(state);
+    expect(isEnabled).toBe(false);
+  });
+
+  it('execute with dispatch', () => {
+    const state = EditorState.create({ schema: schema1 });
+    command.execute(state, dispatch);
```
Comment (Tezra) [inline src/TextAlignCommand.test.ts:124]: missing expect

File: src/TextAlignCommand.test.ts [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+  });
+  it('execute without dispatch', () => {
+    const state = EditorState.create({ schema: schema1 });
+    command.execute(state);
```
Comment (Tezra) [inline src/TextAlignCommand.test.ts:128]: missing expect

File: src/TextLineSpacingCommand.ts [hunk 6]
```diff
@@ -124,22 +125,23 @@ class TextLineSpacingCommand extends UICommand {
...
+    dispatch = dispatch || noop;
     const { schema, selection } = state;
     const tr = setTextLineSpacing(
       state.tr.setSelection(selection),
       schema,
       this._lineSpacing
     );
     if (tr.docChanged) {
-      dispatch && dispatch(tr as Transaction);
+      dispatch(tr as Transaction);
```
Comment (Tezra) [inline src/TextLineSpacingCommand.ts:144]: Typescript has a null safe noop. Replace all instances of noop with dispatch?.(tr)

File: src/ui/PointerSurface.test.tsx [hunk 7]
```diff
@@ -0,0 +1,171 @@
...
+
+
+declare let describe: jest.Describe;
+declare let it: jest.It;
+declare const expect: jest.Expect;
+
+
+describe('PointerSurface', () => {
+  it('renders without crashing', () => {
+    shallow(<PointerSurface />);
```
Comment (Tezra) [inline src/ui/PointerSurface.test.tsx:17]: missing expect

Comment (Tezra) [reply src/ui/PointerSurface.test.tsx:17]: There is an expect for no errors that will give clearer test failure messages

File: src/ui/PopUp.test.tsx [hunk 8]
```diff
@@ -0,0 +1,148 @@
...
+    wrapper = shallow(
+      <PopUp
+        View={MockView}
+        close={mockClose}
+        popUpParams={mockPopUpParams}
+        viewProps={mockViewProps}
+      />
+    );
+  });
+
```
Comment (Tezra) [inline src/ui/PopUp.test.tsx:65]: double beforeEach and afterEach. merge them.

**Review by Tezra — CHANGES_REQUESTED** [review]

**Review by Tezra — CHANGES_REQUESTED** [review]
Tests failing "Schema is missing its top node type ('doc')"

**Review by Tezra — APPROVED** [review]

**Review by rodgersgb — APPROVED** [review]

**Pull request merged**